packages/labs/router/src/routes.ts

```diff
@@ -215,8 +223,17 @@ export class Routes implements ReactiveController {
   /**
    * Matches `url` against the installed routes and returns the first match.
    */
-  private _getRoute(pathname: string) {
-    return this.routes.find((r) => getPattern(r).test({pathname: pathname}));
+  private _getRoute(pathname: string): RouteConfig | undefined {
+    const matchedRoute = this.routes.find((r) =>
+      getPattern(r).test({pathname: pathname})
+    );
+    if (matchedRoute || this.fallbackRoute === undefined) {
+      return matchedRoute;
+    }
+    if (getPattern(this.fallbackRoute).test({pathname: pathname})) {
```

Comment:
I think the fallback shouldn't be able to have a pattern.

Reply:
Done!